src/plugins/fcrypt/fcrypt.c

```diff
@@ -244,18 +249,21 @@ static int fcryptGpgCallAndCleanup (Key * parentKey, KeySet * pluginConfig, char
 		shredTemporaryFile (tmpFileFd, parentKey);
 		if (unlink (tmpFile))
 		{
-			ELEKTRA_ADD_WARNINGF (ELEKTRA_WARNING_FCRYPT_UNLINK, parentKey, "Affected file: %s, error description: %s", tmpFile,
-					      strerror (errno));
+			ELEKTRA_ADD_RESOURCE_WARNINGF (
+				parentKey,
+				"Failed to unlink a temporary file. WARNING: unencrypted data may leak! Please try to delete "
+				"the file manually. Affected file: %s, error description: %s",
```

Comment:
Is not according to our rules (should be Reason: instead of error description:)?